database/seeders/CreateAdminUserSeeder.php

```diff
@@ -32,5 +32,19 @@ public function run()
         $role->syncPermissions($permissions);
 
         $user->assignRole([$role->id]);
+
+        $role2 = Role::create(['name' => 'super-admin']);
+        // gets all permissions via Gate::before rule; see AuthServiceProvider
+        // create demo users
+        $user = User::create([
```

Comment:
Why need to super admin gate user role? Already have an admin role. So no need to add a super admin role from the seed. If you need it you can create it from the admin panel & assign it to a user.

Reply:
ok, that make sense.